[2024-08-28T03:19:13.236+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-28T03:19:13.261+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: country_data_pipeline.extract_data manual__2024-08-28T03:19:02.815631+00:00 [queued]>
[2024-08-28T03:19:13.278+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: country_data_pipeline.extract_data manual__2024-08-28T03:19:02.815631+00:00 [queued]>
[2024-08-28T03:19:13.279+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-28T03:19:13.330+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): extract_data> on 2024-08-28 03:19:02.815631+00:00
[2024-08-28T03:19:13.336+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=413) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-28T03:19:13.336+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'country_data_pipeline', 'extract_data', 'manual__2024-08-28T03:19:02.815631+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dags.py', '--cfg-path', '/tmp/tmpvjmvbdyc']
[2024-08-28T03:19:13.337+0000] {standard_task_runner.py:72} INFO - Started process 422 to run task
[2024-08-28T03:19:13.338+0000] {standard_task_runner.py:105} INFO - Job 45: Subtask extract_data
[2024-08-28T03:19:13.439+0000] {task_command.py:467} INFO - Running <TaskInstance: country_data_pipeline.extract_data manual__2024-08-28T03:19:02.815631+00:00 [running]> on host 1ba7238dfcc0
[2024-08-28T03:19:13.582+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='country_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T03:19:02.815631+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-28T03:19:02.815631+00:00'
[2024-08-28T03:19:13.584+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-28T03:19:13.741+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-28T03:19:13.757+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-28T03:19:13.758+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=country_data_pipeline, task_id=extract_data, run_id=manual__2024-08-28T03:19:02.815631+00:00, execution_date=20240828T031902, start_date=20240828T031913, end_date=20240828T031913
[2024-08-28T03:19:13.834+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-28T03:19:13.895+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-28T03:19:13.918+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
